README.md

```diff
@@ -64,9 +64,9 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
```

Comment:
Should double check whether or not this is actually unnecessary.

Reply:
Verified this in a private repository and it is indeed unnecessary (for this use case).